docs/representing_statistics.md

```diff
@@ -0,0 +1,127 @@
+
+The first four are defined in Schema.org, and the latter four are Data Commons
+extensions.
+
+`Curator` and `Provenance` are necessary for representing who published
+the data to Data Commons and where the data is sourced from.
+`StatisticalVariable` and `StatVarObservation` are relatively new types,
+introduced to reduce the data import and usage complexities of
+`StatisticalPopulation` and `Observation` schema.
+
+## Intro to `StatisticalVariable` and `StatVarObservation`
+
+[`StatisticalVariable`](https://browser.datacommons.org/kg?dcid=StatisticalVariable)
+represents any type of statistical metric that can be measured at a place and
+time. Some examples include: median income, median income of females,
+number of high school graduates, unemployment rate,
+prevalence of diabetes, essentially anything you might call a metric,
+statistic, or measure.
+
+[`StatVarObservation`](https://browser.datacommons.org/kg?dcid=StatVarObservation)
+represents an actual measurement of a `StatisticalVariable` in a given place and time.
+
+The statement "According to the US Census ACS 5 Year Estimates,
+the median age of people in San Antonio, Texas in 2014 was 39.4."
+can be represented as:
+
+```
+Node: Observation_MedianAge_SanAntonio_TX_2014
+typeOf: dcs:StatVarObservation
+variableMeasured: dcid:MedianAge
+observationAbout: dcid:geoId/4865000
+observationDate: "2014"
+value: 39.4
+```
+
+Where [`MedianAge`](https://browser.datacommons.org/kg?dcid=MedianAge)
+is a `StatisticalVariable` schema node that only needs to be defined once.
+
+
+## Mapping `StatisticalVariable` and `StatVarObservation` to `StatisticalPopulation` and `Observation`
+
+The information encoded in `Observation_MedianAge_SanAntonio_TX_2014`
+and `MedianAge` are sufficient for translating into `StatisticalPopulation`
+and `Observation` representations, and we go through this exercise to
+illustrate the value of `StatisticalVariable` and `StatVarObservation`.
+
+For convenience, we copy the MCF for
+[`MedianAge`](https://browser.datacommons.org/kg?dcid=MedianAge) here:
+
+```
+Node: dcid:MedianAge	
+typeOf: dcs:StatisticalVariable
+measuredProperty: dcs:age
+measurementMethod: dcs:CensusACS5yrSurvey
+populationType: schema:Person
+statType: dcs:medianValue
+unit: dcs:Year
+```
+
+The `StatisticalPopulation` extracts the `populationType` and `observationAbout` values
+from the `StatVarObservation` for its own `populationType` and `location` values.
+
+```
+Node: StatisticalPopulation_People_SanAntonio_TX
+typeOf: dcid:StatisticalPopulation
+populationType: dcid:Person
+location: dcid:geoId/4865000
+```
+
+The `Observation` copies the `StatisticalVariable`'s `measuredProperty`,
+`measurementMethod`, `measurementQualifier`, `unit`, etc.
+and the `StatVarObservation`'s `observationDate`. It also extracts the
+`statType`value from the `StatisticalVariable` and the `value` value from
+the `StatVarObservation` as its own `.*Value` property and value.
```

Comment:
This last paragraph is a bit confusing, too many **values** in my opinion.

Reply:
Did a little shuffling, hopefully this helps.